src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -217,6 +217,7 @@
       --pf-c-menu-toggle--m-expanded--Color: var(--pf-c-menu-toggle--m-plain--m-expanded--Color);
 
       display: inline-block;
+      color: var(--pf-c-menu-toggle__toggle-icon--Color);
```

Comment:
the var `--pf-c-menu-toggle__toggle-icon--Color` is really for the `.pf-c-menu-toggle__toggle-icon` element, which is the little arrow icon in the toggle. We manipulate that color on hover/focus/expanded in the `.pf-m-plain.pf-m-text` variant to give the toggle some kind of indication of its state, since it doesn't have the bottom border or a background color like other toggle variations.

The disabled plain example is the wrong color, too. 

<img width="201" alt="Screen Shot 2022-03-01 at 1 24 21 PM" src="https://user-images.githubusercontent.com/35148959/156234977-b1f0c50b-f95f-46cf-bfc1-81204d9bcc13.png">

It should be a lighter grey, like the disabled kebab (plain) dropdown toggle

<img width="395" alt="Screen Shot 2022-03-01 at 1 24 35 PM" src="https://user-images.githubusercontent.com/35148959/156235027-29d5914a-fbf2-4f57-8bc4-04a145a3170e.png">